.github/workflows/build_and_test.yml

```diff
@@ -120,6 +120,41 @@ jobs:
       - name: Run mypy to check types
         run: poetry run mypy .
 
+  check-submodules:
```

Comment:
If we want this check to block the merge, we need to add it aa a dependency to `conclusion` job:

https://github.com/neondatabase/neon/blob/723c0971e818848696984fd66c562c9d0cbff948/.github/workflows/build_and_test.yml#L1158-L1168

Reply:
Added